aws_lambda_builders/workflows/java_gradle/workflow.py

```diff
@@ -51,18 +54,30 @@ def __init__(self,
         ]
 
     def get_resolvers(self):
-        return [GradleResolver(self.source_dir, self.os_utils)]
+        gradlew_resolver = GradlewResolver(binary=self._wrapper_name(),
```

Comment:
hang on, why are you using internal `_wrapper_name`? I don't quite understand the interaction between this and GradlewResolver

Reply:
The resolver needs to know whether it's looking for `gradlew.bat` or `gradlew` based on the OS.